.eslintrc.json

```diff
@@ -16,7 +16,10 @@
       "eol-last": "off",
       "arrow-parens": "off",
       "indent": [2, "tab"],
-      "no-tabs": 0
+      "no-tabs": 0,
+      "import/no-cycle": "off",
+      "no-restricted-globals": "off",
+      "no-alert": "off"
```

Comment:
- [ ] You shouldn't change the linters config rules, as those rules are shared by a group of other students too. Kindly make sure that your linting rules match the rules [here](https://github.com/microverseinc/linters-config/tree/master/javascript).

Reply:
Please share screenshot. I'm not seeing any issue